packages/jss/src/StyleSheet.js

```diff
@@ -118,6 +128,17 @@ export default class StyleSheet {
     return rule
   }
 
+  addRule(name: string, decl: JssStyle, options?: RuleOptions): Rule | null {
+    return this.addOrReplaceRule(name, decl, false, options)
+  }
+
+  /**
+   * See issue https://github.com/cssinjs/jss/issues/1360
+   */
+  replaceRule(name: string, decl: JssStyle, options?: RuleOptions): Rule | null {
```

Comment:
the idea of adding a new method `replaceRule` was to completely avoid  passing an option ala `isReplace`,  because it adds a bit of indirection

Reply:
Ah, I finally get your point...